### PR TITLE
update nightly to trace-agent master

### PR DESCRIPTION
**Pull request by gbbr**

### CODE REVIEW

**Comment by gbbr** [comment]
Windows build failing due to https://github.com/golang/go/wiki/InvalidFlag
Seems unrelated to this (?). Not sure what the plan of action is.

File: omnibus/config/software/datadog-trace-agent.rb [hunk 1]
```diff
@@ -32,27 +32,29 @@
   gopath = Pathname.new(project_dir) + '../../../..'
   if windows?
     env = {
-        'GOPATH' => gopath.to_path,
-        'PATH' => "#{gopath.to_path}/bin:#{ENV['PATH']}",
-        'WINDRES' => 'true',
+      'GOPATH' => "#{windows_safe_path(gopath.to_path)}",
```
Comment (derekwbrown) [inline omnibus/config/software/datadog-trace-agent.rb]: Please bring along the comment so we know why the windows steps are different from Linux, for next time we have to chase down build breakages.

**Review by derekwbrown — DISMISSED** [review]
LGTM, if you bring the comments forward.

**Comment by gbbr** [comment]
@derekwbrown thanks for looking. Brought back comments.

**Review by derekwbrown — APPROVED** [review]

**Pull request merged**